zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -500,6 +531,7 @@ ZEXTERN int ZEXPORT z_deflatePending OF((z_streamp strm,
         return deflatePending(strm, pending, bits);
     FINISH_WITH_ERR("deflatePending is not supported!");
```

Comment:
Is that an expected behavior from zlib to write to `stderr` by default ?
It looks like a CLI functionality, rather than a system library one.